src/vivarium/examples/boids/neighbors.py

```diff
@@ -43,7 +43,7 @@ def on_time_step(self, event: Event) -> None:
     # Pipeline sources and modifiers #
     ##################################
 
-    def get_neighbors(self, index: pd.Index) -> pd.Series:
+    def get_neighbors(self, index: pd.Index[int]) -> pd.Series[list[int]]:  # type: ignore[type-var]
```

Comment:
This is that someone common pandas error where it doesn't list out the entire line of expected items: 

```
Type argument "list[int]" of "Series" must be a subtype of "str | bytes | date | time | bool | <10 more items>"  [type-var]
```